src/nextjs/setup-ui-packages.ts

```diff
@@ -39,4 +39,9 @@ export function setupUIPackages(
   })
 
   new projen.SampleFile(project, 'tailwind.config.ts', {sourcePath: path.join(assetsDir, 'tailwind.config.ts.sample')})
+
+  // FIXME This following code is a workaround needed until eslint-plugin-tailwindcss adds typescript support for the config.
+  const customConfig = 'tailwind-config.js'
```

Comment:
@quesabe, shouldn't it be the `tailwind.config.js`?

Reply:
@AleksandrIvanenko I did not test with `tailwind.config.js` - maybe it would work, maybe not. The guy who proposed the solution deliberately named it different from default name. I just assumed there might be some issues. If need be, I can investigate further if it gonna work. However since I test on a very small mock project the results are nor very comprehensive. I'd better rely on your judgement, since you can test in a real environment. So, how do you think we should proceed?

Reply:
@quesabe, let's leave it like this then. Merge/publish it and I'll test it tomorrow.